.gitignore

```diff
@@ -0,0 +1,96 @@
+# Created by https://www.gitignore.io/api/macos,gradle,android,eclipse
+# Edit at https://www.gitignore.io/?templates=macos,gradle,android,eclipse
+
+### Android ###
+# Built application files
+*.apk
+*.ap_
+*.aab
+*.dex
+*.class
+.gradle/
+local.properties
+.navigation/
+captures/
+
+*.iml
+
+.externalNativeBuild
+
+### Eclipse ###
+
+.project
+.classpath
+
+### macOS ###
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
+# Thumbnails
+._*
+
+# Files that might appear in the root of a volume
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.com.apple.timemachine.donotpresent
+
+# Directories potentially created on remote AFP share
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+
+### Gradle ###
+.gradle
+/build/
+
+# Ignore Gradle GUI config
+gradle-app.setting
+
+# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
+!gradle-wrapper.jar
+
+# Cache of project
+.gradletasknamecache
+
+**/build/
+
+# User-specific stuff
+.idea
+
+# File-based project format
+*.iws
+
+# mpeltonen/sbt-idea plugin
+.idea_modules/
+
+# JIRA plugin
+atlassian-ide-plugin.xml
+
+# Cursive Clojure plugin
+.idea/replstate.xml
+
+# Crashlytics plugin (for Android Studio and IntelliJ)
+com_crashlytics_export_strings.xml
+crashlytics.properties
+crashlytics-build.properties
+fabric.properties
+
+*.iml
+modules.xml
+*.ipr
+
+/config/
+
+# Jekyll
+/_site/
```

Comment:
Just minor thing, better to have blank line end of file. The same for Gemfile.

Reply:
Thanks, I updated this.

Also, I added one more setting for Jekyll: `toc_levels`.